src/components/BackLink/BackLink.tsx

```diff
@@ -0,0 +1,28 @@
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+
+import { Container } from '../Container/Container'
+
+export function BackLink({ isPreviewMode }: { isPreviewMode: boolean }) {
+  const router = useRouter()
+  const { pathname } = router
+
+  if (isPreviewMode || !pathname.startsWith('/providers') || pathname === '/providers') return null
+
+  return (
+    <div>
```

Comment:
This `<div>` looks unnecessary

Reply:
without `<div>` the backlink is being centered